leisure/leisure.ipynb

```diff
@@ -0,0 +1,254 @@
+    "#### “Run” the cells to see the graphs\n",
+    "Click “Cell” and select “Run All”.<br> This will import the data and run all the code, so you can see this week's data visualization. Scroll to the top after you’ve run the cells.<br> \n",
+    "\n",
+    "![instructions](https://github.com/callysto/data-viz-of-the-week/blob/main/images/instructions.png?raw=true)\n",
+    "\n",
+    "**You don’t need to do any coding to view the visualizations**.\n",
+    "The plots generated in this notebook are interactive. You can hover over and click on elements to see more information. \n",
+    "\n",
+    "Email contact@callysto.ca if you experience issues."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### About this Notebook\n",
+    "\n",
+    "Callysto's Weekly Data Visualization is a learning resource that aims to develop data literacy skills. We provide Grades 5-12 teachers and students with a data visualization, like a graph, to interpret. This companion resource walks learners through how the data visualization is created and interpreted by a data scientist. \n",
+    "\n",
+    "The steps of the data analysis process are listed below and applied to each weekly topic.\n",
+    "\n",
+    "1. Question - What are we trying to answer? \n",
+    "2. Gather - Find the data source(s) you will need. \n",
+    "3. Organize - Arrange the data, so that you can easily explore it. \n",
+    "4. Explore - Examine the data to look for evidence to answer the question. This includes creating visualizations. \n",
+    "5. Interpret - Describe what's happening in the data visualization. \n",
+    "6. Communicate - Explain how the evidence answers the question. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Question\n",
+    "How much time do Canadians spend playing video games and how does this change with demographics? We will use official Statistics Canada data to examine this question.\n",
+    "\n",
+    "\n",
+    "### Goal\n",
+    "Our goal is to create a series of histograms to observe how much time Canadians spend gaming."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Gather\n",
+    "\n",
+    "The code below will import the Python programming libraries we need to gather and organize the data to answer our question."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import plotly.express as px #used to create interactive plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code below creates lists data from [this 2010 StatsCan table](https://www150.statcan.gc.ca/n1/pub/89-647-x/2011001/tbl/tbl31-eng.html). The same study was done more recently in 2015. However, the more recent time use survey did not ask about video games.\n",
+    "\n",
+    "Our lists our as follows:\n",
```

Comment:
are

Reply:
One day my monolingual self will master this one language.